tests/test_proof.py

```diff
@@ -1,55 +1,27 @@
-from pathlib import Path
-from subprocess import run
 from bs4 import BeautifulSoup
 import pytest
 import os
```

Comment:
minor edit suggestion: 'os' imported but unused

Reply:
Good catch @najuzilu will remove this and the above one.